go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -175,18 +175,31 @@ func processActiveScenarioUpdate() {
 					mnc := ""
 					mcc := ""
 					cellId := ""
+					erabIdValid := false
 					if domain.CellularDomainConfig != nil {
 						mnc = domain.CellularDomainConfig.Mnc
 						mcc = domain.CellularDomainConfig.Mcc
+						cellId = domain.CellularDomainConfig.DefaultCellId
 					}
-					if poa.Poa4GConfig != nil {
-						if poa.Poa4GConfig.CellId != "" {
-							cellId = poa.Poa4GConfig.CellId
-						} else {
-							cellId = domain.CellularDomainConfig.DefaultCellId
+					switch poa.Type_ {
+					case mod.NodeTypePoa4G:
+						if poa.Poa4GConfig != nil {
+							if poa.Poa4GConfig.CellId != "" {
+								cellId = poa.Poa4GConfig.CellId
```

Comment:
What happens in the rest of the code if the scenario does not have a CellId set?
- should use what is set else set a well-known hardcoded default value (ex. 0000000 or FFFFFFF)
- same applies for 5G.